vassal-app/src/main/java/VASSAL/tools/imports/adc2/ADC2Module.java

```diff
@@ -713,7 +714,7 @@ public String getClassName() {
     }
 
     protected void setValue(int index, String value) {
-      byte[] b = value.getBytes();
+      byte[] b = value.getBytes(StandardCharsets.UTF_8);
```

Comment:
Does anything call this method?

Reply:
No callers in Vassal, perhaps in module code?

Reply:
This is in the AdC2 converter. No module should be calling into this.

Reply:
This method can be removed.